lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -170,7 +170,7 @@ window.runLighthouseInWorker = function(port, url, options, categoryIDs) {
  */
 window.createReportPageAsBlob = function(results, reportContext) {
   performance.mark('report-start');
-
+  filterOutArtifacts(results);
```

Comment:
did you mean to include this second one? `runLighthouseInExtension` would run this twice if it's here too